roles/openshift_cluster_monitoring_operator/files/cluster-monitoring-config.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: ConfigMap
```

Comment:
This probably also needs to be a template.openshift.io/v1.Template and the images should be configurable

Reply:
Separate template, or the same template as the Deployment?